heroic-core/src/main/java/com/spotify/heroic/shell/task/MetadataWrite.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2017 Spotify AB.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.heroic.shell.task;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.spotify.heroic.common.DateRange;
+import com.spotify.heroic.common.Series;
+import com.spotify.heroic.dagger.CoreComponent;
+import com.spotify.heroic.metadata.MetadataManager;
+import com.spotify.heroic.metadata.WriteMetadata;
+import com.spotify.heroic.shell.AbstractShellTaskParams;
+import com.spotify.heroic.shell.ShellIO;
+import com.spotify.heroic.shell.ShellTask;
+import com.spotify.heroic.shell.TaskName;
+import com.spotify.heroic.shell.TaskParameters;
+import com.spotify.heroic.shell.TaskUsage;
+
+import eu.toolchain.async.AsyncFuture;
+
+import org.kohsuke.args4j.Option;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import dagger.Component;
+import lombok.ToString;
+
+@TaskUsage("Write metadata")
+@TaskName("metadata-write")
+public class MetadataWrite implements ShellTask {
+
```

Comment:
Generally we only use blank lines between member and methods of a class, which should be in line with the Google code style.